static/styles/message.css

```diff
@@ -60,6 +85,16 @@ $distance_of_text_elements_from_message_box_top: 8px;
                     margin-top: 3px;
                 }
             }
+
+            &.content_edit_mode {
+                .is_me_message {
+                    & ~ .alert-msg,
+                    & ~ .message_time,
+                    & ~ .message_controls {
+                        grid-row: 1 / 2;
```

Comment:
This probably deserves a comment for what this `grid-row` thing is doing.

Reply:
Actually, I have a different thought -- I wonder whether we use one of those tools that make nice ASCII diagrams, and put the diagram in a comment at the top of the `message_row` section showing the structure of message rows with all the regions labeled, and then make a separate one for `is_me_message`.

Idunno, maybe that's not helpful, just trying to think about how to make this readable.

Reply:
That's a nice idea. Added the ASCII diagrams.